### PR TITLE
Code Done successfully

### PR DESCRIPTION
**Pull request by noor848**

### CODE REVIEW

**Comment by noor848** [comment]
https://leetcode.com/problems/merge-two-sorted-lists/

File: Task4.cs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        }
+        public static void print(ListNode head)
+        {
+            while (head != null)
+            {
+                Console.Write(head.val + " ");
+                head = head.next;
+            }
+        }
+        public static void Main(string[] args)
```
Comment (hayatzsala) [inline Task4.cs]: @noor848 same as before

File: Task4.cs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            if (l2 == null)
+            {
+                l3.next = l1;
+            }
+
+            return list3.next;
+
+        }
+        public static ListNode generateLinkList(int[] arr)
+        {
```
Comment (hayatzsala) [inline Task4.cs:63]: ❤️ love it

File: Task4.cs [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        public ListNode(int val = 0, ListNode next = null)
+        {
+            this.val = val;
+            this.next = next;
+        }
+    }
+    public class Task4
+    {  public static ListNode MergeTwoLists(ListNode list1, ListNode list2)
+        {
+            ListNode list3 = null, l3 = null;
```
Comment (hayatzsala) [inline Task4.cs]: CLEAN CODE: declare the l3 in new line

**Review by hayatzsala — APPROVED** [review]

**Comment by hayatzsala** [comment]
@noor848 solve the conflict please,
and why this conflict ? do u solve the 2 problems in same project?

**Review by hayatzsala — APPROVED** [review]

**Pull request merged**